kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/data/auto_conf.yaml

```diff
@@ -1,35 +1,44 @@
+## @param ad_identifiers - list of strings - required
+## A list of container identifiers that are used by Autodiscovery to identify
+## which container the check should be run against. For more information, see:
+## https://docs.datadoghq.com/agent/guide/ad_identifiers/
+#
 ad_identifiers:
   - kube-apiserver
 
+## All options defined here are available to all instances.
+#
 init_config:
 
+## Every instance is scheduled independent of the others.
+#
 instances:
+
     ## @param prometheus_url - string - required
     ## The URL where your application metrics are exposed by Prometheus.
     #
-  - prometheus_url: "https://%%host%%:6443/metrics"
+  - prometheus_url: https://%%host%%:6443/metrics
 
-    ## @param bearer_token_auth - string - optional
+    ## @param bearer_token_auth - boolean - optional - default: true
     ## Used if you are using RBACs and need the Agent to authenticate
     ## against the APIServer to retrieve metrics. Default to true.
     #
-    bearer_token_auth: true
+    # bearer_token_auth: true
 
-    ## @param bearer_token_path - string - optional
+    ## @param bearer_token_path - string - optional - default: /var/run/secrets/kubernetes.io/serviceaccount/token
     ## Used to specify the path where the service account token is located.
-    ## Default to "/var/run/secrets/kubernetes.io/serviceaccount/token".
     #
-    # bearer_token_path: "/var/run/secrets/kubernetes.io/serviceaccount/token"
+    # bearer_token_path: /var/run/secrets/kubernetes.io/serviceaccount/token
 
-    ## @param ssl_verify - string - optional
-    ## Used to verify self signed certificates. Default to false.
+    ## @param ssl_verify - boolean - optional - default: False
+    ## Used to verify self signed certificates.
     #
     # ssl_verify: false
 
-    ## @param tags - list of key:value element - optional
+    ## @param tags - list of strings - optional
     ## List of tags to attach to every metric, event and service check emitted by this integration.
     ##
     ## Learn more about tagging: https://docs.datadoghq.com/tagging/
     #
-    tags:
-      - "apiserver:%%host%%"
+    # tags:
+    #   - apiserver:%%host%%
```

Comment:
should I do something about this?

Reply:
IMO we should keep the current behaviour. Commenting it would be a breaking change.

There is `enabled:true` that can be used to keep the current behaviour: https://github.com/DataDog/integrations-core/blob/a8e1df5e5e2af0007d49de7741c4184eb3fa7fef/datadog_checks_dev/tests/tooling/configuration/consumers/test_example.py#L1247

Reply:
Seems it does not work